.github/workflows/build.yml

```diff
@@ -65,6 +67,14 @@ jobs:
           mkdir -p ./jacoco
           echo $PR_NUMBER > ./jacoco/jacoco_report_number
           echo '![Code Coverage]'"(https://img.shields.io/badge/Code%20Coverage-${coverage}%25-${color}?style=flat)" > ./jacoco/jacoco_report_badge
+
+      - name: Deploy Edge+Backend assets
+        if: success() && github.event_name == 'push' && github.ref_name == 'develop'
+        run: ./gradlew buildEdge buildBackend release -DmavenPublishRepository=RemoteMavenRepository
+        env:
+          CI_DEPLOY_USERNAME: ${{ secrets.CI_DEPLOY_USERNAME }}
+          CI_DEPLOY_PASSWORD: ${{ secrets.CI_DEPLOY_PASSWORD }}
```

Comment:
This build yml runs on every push to every pull-request. Is it sensible to deploy every time?